doc/Type/independent-routines.pod6

```diff
@@ -93,19 +96,30 @@ EVAL "use v5.20; say 'Hello from perl!'", :lang<Perl5>;
 You need to have L<C<Inline::Perl5>|https://github.com/niner/Inline-Perl5> for
 this to work correctly.
```

Comment:
What about Inline-Python?

Reply:
You would like to have EVAL compatible packages listed here ? Or may be a link to https://modules.raku.org/search/?q=inline ?

Reply:
Well, going forward it's probably the best thing... We can't change this every time a new Inline:: comes up.

Reply:
Good for me. So would 4687964 be OK ?